src/nodes/html.ts

```diff
@@ -260,7 +261,7 @@ export default class HTMLElement extends Node {
 					// Whitespace node, postponed output
 					currentBlock.prependWhitespace = true;
 				} else {
-					let text = node.text;
+					let text = (<TextNode>node).trimmedText;
```

Comment:
This reinstates the trim functionality (prior to 3.3.2) with our special trim, which should produce a cleaner output.